oak-core/src/main/java/org/apache/jackrabbit/oak/query/ast/FullTextSearchScoreImpl.java

```diff
@@ -118,4 +118,9 @@ public OrderEntry getOrderEntry(SelectorImpl s, OrderingImpl o) {
         return null;
     }
 
+    @Override
+    public String getOrderEntryPropertyName(SelectorImpl s) {
+        return null;
```

Comment:
Order by score is possible... I think it is always ignored right now, but that's in implementation. If it always returns null, then the logic in QueryImpl would throw an exception (NPE right now). I think that's not correct. Instead, it's probably better to return the name of the score pseudo-property here.

Reply:
Actually getOrderEntryPropertyName basically is needed to compare the property name set for the Ordering any particular implementation of DynamicOperand - In case of FullTextSearchScoreImpl - no Order is set , so I thought maybe it would be better to return null from getOrderEntryPropertyName  too.

But I can return score pseudo-property here .